From ccd4e2f078d00fa4570d2bd56802c726286d1020 Mon Sep 17 00:00:00 2001
From: Martin Kolman <mkolman@redhat.com>
Date: Wed, 10 Oct 2018 17:12:01 +0200
Subject: [PATCH] Set help id for the OSCAP addon provided spoke (#1638068)

The new Anaconda help system now operates on help ids instead of pointing to
individual files.

So drop the old property and replace it with a proper help_id.

Resolves: rhbz#1638068
---
 org_fedora_oscap/gui/spokes/oscap.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/org_fedora_oscap/gui/spokes/oscap.py b/org_fedora_oscap/gui/spokes/oscap.py
index d9fe548..36fd656 100644
--- a/org_fedora_oscap/gui/spokes/oscap.py
+++ b/org_fedora_oscap/gui/spokes/oscap.py
@@ -179,8 +179,8 @@ class OSCAPSpoke(NormalSpoke):
     # name of the .glade file in the same directory as this source
     uiFile = "oscap.glade"
 
-    # name of the file providing help content for this spoke
-    helpFile = "SecurityPolicySpoke.xml"
+    # id of the help content for this spoke
+    help_id = "SecurityPolicySpoke"
 
     # domain of oscap-anaconda-addon translations
     translationDomain = "oscap-anaconda-addon"
